src/gui/windows/debug.zig

```diff
@@ -61,6 +66,14 @@ pub fn render() void {
 		y += 8;
 		draw.print("Queue size: {}", .{main.threadPool.queueSize()}, 0, y, 8, .left);
 		y += 8;
+		const perf = main.threadPool.performance.read();
+		const values = comptime std.enums.values(TaskType);
+		for(values) |t| {
+			const name = @tagName(t);
+			const i = @intFromEnum(t);
+			draw.print("    {s} time: {} ms/frame ({} µs/task)", .{name, @divFloor(perf.utime[i], 1000), @divFloor(perf.utime[i], @max(1, perf.tasks[i]))}, 0, y, 8, .left);
```

Comment:
On that note, is this is even a useful metric? The time per tasks seems rather unrelated to the time per frame. Coupling the two seems weird.

Reply:
Both are useful- in determining the effect on the user versus benchmarking the task in isolation. Maybe there is a clearer way to format that?

Reply:
But the framerate could fluctuate. Like if you look in a different direction, then the ms/frame metric will be different, even though the actual task performance is the same.
Maybe milliseconds per second would be a more stable unit? But even then the number will get continuously smaller if you don't have any tasks generating. So it still seems kind of useless to me for doing actual measurements.

Reply:
Well, it does at least tell you where to look. If some tasks are very expensive but very rare, it might be worth checking how it is overall.
Maybe we should use %time spent on this task rather than ms/s? As a proxy to cpu usage? In any case I believe it _should_ approach zero when the tasks decrease, to signal that the developer need not optimize it further for the current scene.

Reply:
Oh, then I think percentage of total task time (sum over the times of all task types) would be best. Then the percentages would always add up to 100%, giving us a clearer picture than the total system time would.